qemu/tests/cfg/win_virtio_update.cfg

```diff
@@ -16,11 +16,12 @@
     image_size_stg = 1G
     drive_format_stg = virtio
     force_create_image_stg = yes
-    nics += " nic2"
+    nics = " nic2"
```

Comment:
Why remove the default  nic?
we can't make the guest has the nic driver already.